app/models/claim.rb

```diff
@@ -138,7 +128,7 @@ def submittable?
   end
 
   def approvable?
-    submitted? && approved_at.nil?
+    submitted? && check.nil?
```

Comment:
This should be checking that the `check` is also not approved, right?

Reply:
This was one of the balance ones from me again. My thinking was that a check can only mean an approval at this point so no check means it is not approved.